SDK/Directory.Build.props

```diff
@@ -4,7 +4,7 @@
     <Copyright>Microsoft Corp. All rights reserved.</Copyright>
     <Company>Microsoft Corporation</Company>
     <!-- Pin C# version. C# 8 features might have issues on Mono runtime. -->
-    <LangVersion>7.0</LangVersion>
+    <LangVersion>8.0</LangVersion>
```

Comment:
The comment for this parameter says that version 8 may cause problems with Mono runtime. Has this changed?

Reply:
Now it doesn't even build with lang version 7 😅